### PR TITLE
Kogito 1.4.1.Final + OptaPlanner 8.4.1.Final

### PR DESCRIPTION
**Pull request by evacchi**

### CODE REVIEW

**Comment by evacchi** [comment]
it looks like it is this check https://github.com/kiegroup/kogito-runtimes/blob/master/kogito-codegen-modules/kogito-codegen-api/src/main/java/org/kie/kogito/codegen/api/utils/AppPaths.java#L90

**Comment by evacchi** [comment]
hotfix on the way -- 1.4.1

File: integration-tests/kogito/invoked/root/rpkgtests/pom.xml [hunk 1]
```diff
@@ -14,10 +14,25 @@
...
+        <dependency>
+            <groupId>org.kie.kogito</groupId>
+            <artifactId>kogito-quarkus-decisions-integration-test</artifactId>
+            <version>1.4.1.Final</version>
+        </dependency>
         <dependency>
             <groupId>org.kie.kogito</groupId>
             <artifactId>kogito-quarkus-integration-test</artifactId>
-            <version>1.3.0.Final</version>
+            <version>1.4.1.Final</version>
```
[the remark this answers is not in the document]
Comment (evacchi) [reply integration-tests/kogito/invoked/root/rpkgtests/pom.xml:25]: I think that is code-generated with `mvn -Pregen-kogito validate -N` -- all I did was updating `/pom.xml` with relevant `<testJar>` sections

[the remark this answers is not in the document]
Comment (aloubyansky) [reply integration-tests/kogito/invoked/root/rpkgtests/pom.xml:25]: Ah, right, ok then.

**Comment by evacchi** [comment]
@gsmet all green here :)

**Review by gsmet — APPROVED** [review]

**Pull request merged**